src/swell/tasks/run_jedi_local_ensemble_da_executable.py

```diff
@@ -227,12 +226,16 @@ def execute(self) -> None:
                 self.logger.info('Running ensmean_only')
                 run_executable(self.logger, self.cycle_dir(), np,
                                jedi_ensmeanvariance_executable_path,
-                               jedi_config_file, output_log_file)
+                               jedi_config_file, output_log_file, perhost=perhost)
             else:
-                self.logger.info('Running '+jedi_executable_path+' with '+str(np)+' processors.')
                 run_executable(self.logger, self.cycle_dir(), np, jedi_executable_path,
-                               jedi_config_file, output_log_file)
+                               jedi_config_file, output_log_file, perhost=perhost)
         else:
+            mpi_command = "mpirun"
+            if not (perhost is None or perhost == "None"):
```

Comment:
`perhost` is configured as an integer or `None`. It should never be a string, so this check is unnecessary.

```suggestion
            if perhost is not None:
```

Reply:
It is perplexing to me as well: 
`if perhost is not None:` does not work at here, only
`if not (perhost is None or perhost == "None"):`  works. 
So there is a bug somewhere. 
I found this line from @rtodling's PR for perhost, and it worked here, the other way fails.

Reply:
Yeah, this was my "quick fix" because @rtodling's PR failed.

Reply:
OK -- let's keep it as is for now, and I'll open an issue.

Reply:
I think this is a bug assigning None to an integer,  GPT wrote
```
x = 10
x = None  # Allowed, but x is no longer an integer
```

With the code from previous PR:

https://github.com/GEOS-ESM/swell/blob/39f35f8179a51d3386c29f59619c09dfae56f6bc/src/swell/tasks/task_questions.yaml#L833-L841

The `perhost` default is a "None" string, which costed me an hour to wonder why the `if perhost is not None` does not work here.   We should give default 0 for perhost.

Reply:
The issue here is that `yaml` doesn't know to treat the string `None` as a special value; it just reads it as a string, `"None"`. The best solution is to replace `default_value: None` with `default_value: null` (or, less obviously, a blank entry -- `default_value: `; or even less obviously, a `~`, which yaml interprets as empty -- `default_value: ~`), which Python will read in as a `None` object.

```python
import yaml

doc1 = """
a: 1
b:
    b1: null
    b2: yes
"""

d1 = yaml.safe_load(doc1)
d1["b"]["b1"] is None
# True

doc2 = """
a: 1
b:
    b1:
    b2: yes
"""

d2 = yaml.safe_load(doc2)
d2["b"]["b1"] is None
# True


doc3 = """
a: 1
b:
    b1: None
    b2: yes
"""
d3 = yaml.safe_load(doc3)
d3["b"]["b1"] is None
# False -- returns the string "None"
```

The _bigger_ issue is that we don't have a good way of catching these kinds of parsing/type errors at runtime (and it's even tricky to do with static type checking). Which is all the more reason I'm looking forward to #514 --- once these things are Python objects, we pick up a lot more power from static and runtime checking.

Reply:
In any case, this isn't your bug to fix; we can resolve it once this is merged.